core-test/src/test/scala/org/bitcoins/core/crypto/NativeSecp256k1ScalaTest.scala

```diff
@@ -0,0 +1,73 @@
+package org.bitcoins.core.crypto
+
+import org.bitcoin.NativeSecp256k1
+import org.bitcoins.core.util.CryptoUtil
+import org.bitcoins.testkit.core.gen.NumberGenerator
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import scodec.bits.ByteVector
+
+class NativeSecp256k1ScalaTest extends BitcoinSUnitTest {
+  behavior of "NativeSecp256k1"
+
+  it must "act like a normal schnorrSign if the normal nonce is specified" in {
+    forAll(
+      NumberGenerator.bytevector(32).filter(_.toArray.exists(_ != 0.toByte)),
```

Comment:
Yeah, private keys can't be zero in Schnorr sigs because then there's no commitment to the message being signed (i.e. it would be a valid signature for any message)

And I found that there was a private key `Gen` in `CryptoGenerators` so I put it there :)